.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,49 @@
+name: Main Documentation Checks
```

Comment:
this task we have no where else ....

@lengau ?

Reply:
That one is from the [sphinx-docs-starter-pack](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/.github/workflows/automatic-doc-checks.yml)

Reply:
this seems like duplication from what we have in tox, so please remove it

Reply:
It seems the commands are different lints. This one check texts / words / links, not the syntax in tox.

Reply:
I'm mixed about it. I'd like them to be in tox so, but I'd also like to keep as close to upstream as possible.

Reply:
ok, let's get this in as is for now, but they names do not follow our convention either.

@lengau can you please open issues to normalize this into tox?

Reply:
https://github.com/canonical/craft-store/issues/105